islandora_get_csv.drush.inc

```diff
@@ -69,6 +72,9 @@ function drush_islandora_get_csv_get_csv() {
   foreach ($settings_variables as $variable) {
     $options[$variable] = variable_get($variable);
   }
+  if(drush_get_option('include_descendants')) {
+    $options['islandora_get_csv_include_descendents'] = true;
```

Comment:
You need to define this option in the islandora_get_csv_command() function.

Also, can you add a note to the README about how to use this new option?

Reply:
> You need to define this option in the islandora_get_csv_command() function.

Sorry, that's wrong. Please ignore. Will test again when I'm not in a haze.

Reply:
> You need to define this option in the islandora_get_csv_command() function.

Not sure what you mean; isn't it there in line 30?